app/controllers/state_file/state_file_pages_controller.rb

```diff
@@ -41,5 +41,9 @@ def transfer_url(key, redirect_url)
       uri.query = { authorizationCode: key }.to_query
       uri.to_s
     end
+
+    def current_intake
+      @current_intake ||= current_state_file_az_intake || current_state_file_ny_intake
+    end
```

Comment:
A current intake method already exists here from the application controller - I figured it should be customized for statefile